plugin/plugin.go

```diff
@@ -131,7 +131,9 @@ func (p *YdbStorage) initDB(v *viper.Viper) {
 	conn, err := db.DialFromViper(ctx, v,
 		ydb.WithSessionPoolSizeLimit(p.opts.PoolSize),
 		ydb.WithSessionPoolKeepAliveTimeout(time.Second),
-		ydb.WithConnectParams(ydb.EndpointDatabase(p.opts.DbAddress, p.opts.DbPath.Path, false)),
+		ydb.WithEndpoint(p.opts.DbAddress),
+		ydb.WithDatabase(p.opts.DbPath.Path),
+		ydb.WithInsecure(),
```

Comment:
Secure flag is controlled inside `db.DialFromViper` depending whether it's yandex-cloud public installation or not. Shouldn't be here

Reply:
previous version contains `false` as `secure` arg
latest option `WithSecure` replaced inside `db.DialFromViper` old option `WithInsecure` from here
It's ok

Reply:
Then it's just a confusing no-op, still better to clean this up